queries/scala/highlights.scm

```diff
@@ -0,0 +1,52 @@
+;; keywords
+
+[
+  "abstract"
+  "case"
+  "catch"
+  "class"
+  "def"
+  "do"
+  "extends"
+  "final"
+  "finally"
+  "forSome"
+  "implicit"
+  "lazy"
+  "macro"
+  "match"
+  "new"
+  "null"
+  "object"
+  "override"
+  "package"
+  "private"
+  "protected"
+  "sealed"
+  "super"
+  "this"
+  "throw"
+  "trait"
+  "try"
```

Comment:
We have a exception category for try/catch stuff

Reply:
done.